Does not compile without zeroize feature (--no-default-features)

anoncreds-rs supports opt-in to `zeroize` via a default feature flag; which is great. however the crate does not actually compile without the zeroize feature:
```
anoncreds-rs git:(main) cargo build --no-default-features                   
   Compiling anoncreds v0.1.0 (/Users/gmulhearne/Documents/dev/rust/anoncreds-rs)
error[E0432]: unresolved import `zeroize`
 --> src/data_types/credential.rs:2:5
  |
2 | use zeroize::Zeroize;
  |     ^^^^^^^ use of undeclared crate or module `zeroize`

error: cannot determine resolution for the derive macro `Zeroize`
   --> src/data_types/credential.rs:109:39
    |
109 | #[derive(Clone, Debug, PartialEq, Eq, Zeroize, Deserialize, Serialize)]
    |                                       ^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

For more information about this error, try `rustc --explain E0432`.
error: could not compile `anoncreds` (lib) due to 2 previous errors
```

whereas it compiles fine once zeroize is enabled:
```
cargo build --no-default-features --features zeroize
```

--- a/src/data_types/credential.rs
+++ b/src/data_types/credential.rs
@@ -1,4 +1,6 @@
 use std::collections::HashMap;
+
+#[cfg(feature = "zeroize")]
 use zeroize::Zeroize;
 
 use crate::cl::{CredentialSignature, RevocationRegistry, SignatureCorrectnessProof, Witness};
@@ -82,6 +84,7 @@ pub type ShortCredentialValues = HashMap<String, String>;
 #[derive(Clone, Debug, Default, PartialEq, Eq, Deserialize, Serialize)]
 pub struct CredentialValues(pub HashMap<String, AttributeValues>);
 
+#[cfg(feature = "zeroize")]
 impl Drop for CredentialValues {
     fn drop(&mut self) {
         self.zeroize();
@@ -98,6 +101,7 @@ impl Validatable for CredentialValues {
     }
 }
 
+#[cfg(feature = "zeroize")]
 impl Zeroize for CredentialValues {
     fn zeroize(&mut self) {
         for attr in self.0.values_mut() {
@@ -106,7 +110,8 @@ impl Zeroize for CredentialValues {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq, Zeroize, Deserialize, Serialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[cfg_attr(feature = "zeroize", derive(Zeroize))]
 pub struct AttributeValues {
     pub raw: String,
     pub encoded: String,

